test/PubSubTest.cpp

```diff
@@ -27,24 +28,24 @@ TEST(PubSubTest, function_subscription) {
       ASSERT_TRUE(false); // it takes too long. test should fail instead of deadlock
     }
     cmd.set_id(roboteam_utils::Timer::getCurrentTime().count()); // we store the time in the id
-    pub->send("dummy_robotcommand_topic", cmd.SerializeAsString());
+    pub->send(cmd);
   }
 
-  // the communication should be fast (<5ms)
-  EXPECT_LE( roboteam_utils::Timer::getCurrentTime().count() - receivedTime,  5);
+  // the communication should be fast (<10ms)
+  EXPECT_LE( roboteam_utils::Timer::getCurrentTime().count() - receivedTime,  10);
```

Comment:
Is there a reason this is higher? CI server ticks too slow sometimes?

Reply:
Yeah usually it's below 1ms but sometimes it is somewhat higher, causing tests to sometimes fail